3.1/imports/chunk-reading.md

```diff
@@ -68,14 +68,44 @@ class UsersImport implements ToModel, WithBatchInserts, WithChunkReading
 
 ## Keep Track of the Row number
 
-If you need the information about the offset of the chunk you can use the `RemembersChunkOffset` Trait.
+If you need the row number you can use the `RemembersRowNumber` Trait.
 
 ```php
 namespace App\Imports;
 
 use App\User;
 use Maatwebsite\Excel\Concerns\ToModel;
-use Maatwebsite\Excel\Concerns\WithChunkOffset;
+use Maatwebsite\Excel\Concerns\RemembersRowNumber;
+use Maatwebsite\Excel\Concerns\WithChunkReading;
+
+class UsersImport implements ToModel, WithChunkReading
```

Comment:
Can we add a note somewhere that it only works for ToModel?
Also perhaps nice to show of an example with validation? I believe that's what you needed it for?

Reply:
@kenowessels can you pick this comment up?